_site/i18n.js

```diff
@@ -15,9 +16,9 @@
 				return substr; // simple substitution eg {0}
 			}
 			switch(format.shift()) {
-				case "number" : return (new Number(substr)).toLocaleString();
-				case "date" : return (new Date(+substr)).toLocaleDateString(); // date and time require milliseconds since epoch
-				case "time" : return (new Date(+substr)).toLocaleTimeString(); //  eg i18n.text("Key", +(new Date())); for current time
+				case "number" : return (new Number(substr)).toLocaleString(locale);
+				case "date" : return (new Date(+substr)).toLocaleDateString(locale); // date and time require milliseconds since epoch
+				case "time" : return (new Date(+substr)).toLocaleTimeString(locale); //  eg i18n.text("Key", +(new Date())); for current time
```

Comment:
I think as a development tool, if toISOString accepts locale(essentially Timezone), which would be a better choice rather than toLocaleString, but currently toISOString only return the time string based on GMT, which is so sad.